python/print_kotekan_version.py

```diff
@@ -0,0 +1,5 @@
+# Prints the kotekan version using versioneer
+# The script is called from make_version.cmake.in
+from _version import get_versions
+
+print get_versions()["version"]
```

Comment:
This won't work in python 3
```
[  1%] Generating version.c

  File "/Users/jenkins/workspace/kotekan_PR-664/python/print_kotekan_version.py", line 5

    print get_versions()["version"]

                     ^

SyntaxError: invalid syntax
```
(https://lwlab.dunlap.utoronto.ca:18081/blue/organizations/jenkins/kotekan/detail/PR-664/13/pipeline/35)
```suggestion
print(get_versions()["version"])
```